doc/basic_configuration.rst

```diff
@@ -55,17 +61,32 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+Note that you need to have a local smtp service configured for email notifications.
+
+If you do not require email notifications, remove the configuration line entries.
+
 
 Update URLs to point to custom build farm
 -----------------------------------------
 
-Change the Jenkins URL in the entry point yaml to point to your earlier
-provisioned Jenkins master::
+Change the ``rosdistro_index_url`` in the `ros-infrastructure/ros_buildfarm_config <https://github.com/ros-infrastructure/ros_buildfarm_config>`_'s 
```

Comment:
Same as above: I don't think this should reference the one repo used for the ROS buildfarm. The common case will be that a user setting up a custom buildfarm has its own fork of this so the added link isn't really helpful.

Same below another time.

Reply:
Thanks! Fixed.